content/location.md

```diff
@@ -1,38 +1,39 @@
 ---
 title: "Location"
 weight: 9
-draft: true
 menu: main
 type: page
 ---
-The contest will take place at the ...
+The contest will take place at [Education Building 35](https://map.tudelftcampus.nl/poi/education-building-35/) at the Cornelis Drebbelweg. 
 
 
 ## To Delft by Car
 
 * At the A13/E19 between Rotterdam and Den Haag, take exit 10, Delft-Zuid/TU Delft.
-* Follow the Signs TU Delft until you drive on a downward slope until the crossing with the Schoenmakerstraat.
-* Go right on to the Schoenmakerstraat and you have arrived at the campus ring road.
+* Follow the Signs TU Delft until you drive on a downward slope until the crossing with the Schoemakerstraat.
+* Go *left* on to the Schoemakerstraat and you have arrived at the campus ring road.
+* Follow the signs P5 
 
 Parking on the street is paid on all days (except Sunday) from 12:00 until 24:00.
 At campus parking locations, parking is free between Friday 18:00 and Monday 08:00.
 The six large campus parking locations are well signposted on the campus ring road.
-Recommended campus parking locations:
-
-* P Aula (navigation address Van der Waalsweg 1)
-* P EWI, 8-minute walk (navigation address Feldmannweg 2)
-* P Sports, 10-minute walk (navigation address Van den Broekweg 1)
+Recommended campus parking location is [P5 Rotterdamseweg](https://map.tudelftcampus.nl/poi/p-rotterdamseweg/) (navigation address Cornelis Drebbelweg 1, Delft)
 
 
 ## By Public Transport
 Delft has two train stations:
 
 * **Delft**, connecting to most train connections between Rotterdam and Den Haag HS and also has several bus connections to the contest site.
 * **Delft Campus**, connecting only on sprinter-lines, it provides no bus connections, but it is closer to the contest site on foot.
-Planning your trip is possible from the website of the [Dutch Railways](https://www.ns.nl/en/travel-information).
 
-Bus lines 40 (weekdays only), 69 (weekdays only), 174 (weekdays only), and 455 (every day) stop very close to the contest site by bus stop Aula TU.
-Bus tickets can be bought in the bus with Debit Card only.
+Planning your trip is possible from the website of the [Dutch Railways](https://www.ns.nl/en/travel-information). Train tickets can be bought at the machines at
+the station, or  you can check in using [OVPay](https://www.ovpay.nl/en/how-it-works) using a foreign contactless debit card,
+a credit card issued by MasterCard/Maestro and VISA/Vpay, or by phone using Google Pay or Apple Pay.
```

Comment:
Maybe one more clarification, just in case? I have no idea how many credit cards are _not_ contactless nowadays, but you never know 😛 
```suggestion
a contactless credit card issued by MasterCard/Maestro and VISA/Vpay, or by phone using Google Pay or Apple Pay.
```